File: bms.py
Title = "CUSTOMER BANKING"
print("~"*len(Title))
print(Title)
print("~"*len(Title))

balance = float(input("\nPlease enter  balance of your Account: "))
print("=======*=======")
print(balance)
print("=======*=======")

while True:

	option = input("""\nSelect an option you want to perform
		1. Create Account
		2. Withdraw
		3. Deposit
		4. Modify Account
		5. Closing Account \n""")



	while option =='1' :
		title = "Creating Account"
		print("*"*len(title))
		print(title)
		print("*"*len(title))

				# FORMAT = ONLY GIVE INPUT AS NAMES
				# NOT IN NUMBER
				#INPUT = AHDDDHFDG
				#INPUT != 2214433

		while True:
			print("Enter the name of Account Holder")
			acc_holder = input("Acc_holder Name: ")
			if acc_holder[0].isalpha()!=True :
				print("Please Enter name in valid format")

				
			elif acc_holder.isalnum() == False:
				print("Please Enter name in valid format")
			else :
				break;
		while True:
			print("Enter the address of Account Holder")
			address = input("Address: ")
			if address[0].isalpha()!=True :
				print("Please Enter name in valid format")

				
			elif address.isalnum() == False:
				print("Please Enter name in valid format")
			else :
				break;
		while True:
			print("Enter mobile number of the Account Holder")
			mbl_number =  int(input("Mobile Number: "))
			print("Enter your Adhar Number")
			adhar_number =  int(input("Adhar Number: "))
			print("Enter age")
			age =  int(input("age: "))

			choice = input(""" Enter your choice
				1.Savings Account
				2.Current Account\n""")

			while choice == '1' :
				print("you have choosen Savings account")
				print("Congrats! Your account has been created")
				print("Thank you for creation of account in our bank")
				break
			while choice == '2' :
				print("you have choosen Current account")
				print("Congrats! Your account has been created")
				print("Thank you for creation of account in our bank")
				break
			break
		break

	while option == '2':
		print("You have choosen Withdraw")
		print("Enter amount to be withdrawed")
		Withdraw = float(input("Amount: "))

		if Withdraw > balance :
			print("You have insufficient amount in your Account")
		else :

			print(Withdraw)
			balance = balance - Withdraw
			print("Your Account balance is ......")
			print("============*=============")
			print(balance)
			print("============*=============")
		break

	while option == '3':
		
		print("Enter the amount to be deposited")
		Deposit = float(input("Amount: "))

		print(Deposit)
		balance = balance + Deposit
		print("Your Account balance is ......")
		print("==========*=========")
		print(balance)
		print("==========*=========")
		break

	while option == '4':
		title_1 = "MODIFYING ACCOUNT"
		print("="*len(title_1))
		print(title_1)
		print("="*len(title_1))

		

		selection = input("""Enter your selection

			1. Account holder name
			2. Address
			3. Mobile Number
			4. Age
			5. Adhaar Number \n""")

		while selection == '1':
			print("The Account Holder name that need to be modified: ")
			acc_holder = input("Acc_holder Name: ")
			if acc_holder[0].isalpha()!=True :
				print("Please Enter name in valid format")

			elif acc_holder.isalnum() == False:
				print("Please Enter name in valid format")
			else :
				break;

		while selection == '2':
		
			print("Enter the address of Account Holder that need to be modified: ")
			address = input("Address: ")
			if address[0].isalpha()!=True :
				print("Please Enter name in valid format")

			elif address.isalnum() == False:
				print("Please Enter name in valid format")
			else :
				break;
		while selection == '3':

			print("Enter mobile number that need to be modified")
			mbl_number =  int(input("Mobile number: "))
			break

		while selection == '4':
			print("Enter age to be modified")
			age = int(input("Age: "))
			break

		while selection == '5':
			print("Enter Adhaar Number to be modified ")
			adhar_number = int("Adhaar Number: ")
			break

		print("The modified detials are : ")
		print(f"Name: {acc_holder}")
		print(f"Address: {address}")
		print(f"Mobile Number: {mbl_number}")
		print(f"Age: {age}")
		print(f"Adhaar Number: {adhar_number}")
		break

	while option == '5':
		print("Please explain the reason for closing your account")
		reason = input()
		print("please fill the application")
		print('Enter the name: ')
		name = (input())
		print('Enter the age: ')
		age = int(input())
		print('Enter mobile number: ')
		mobile_number =   int(input())
		print("Enter adhar_number: ")
		adhar_numb =  int(input())
		print("your Account has been closed")
		break

File: BMS.cpp
//PROGRAM ON BANKING MANAGEMENT SYSTEM

#include<iostream>
using namespace std;
class BMS
{
	public:
		char  acc_holder[20],address[20];
		long long int mobile_number,adhar_number;
		int age;
		void create_account();
		void display_account_holder();
		void deposit();
		void withdraw();
		void modifying_account();
		void closing_account();
};
void BMS::create_account()
{

	/* 
	FORMAT = ONLY GIVE INPUT AS LETTER FORMAT
	INPUT = AJFHDHFAK
	INPUT!=274252
	*/
	
	cout<<"Enter the necessary details in order to create account details"<<endl;
	cout<<"Enter the name of the account holder"<<endl;
	cin>>acc_holder;

	cout<<"Enter the address of the account holder"<<endl;
	cin>>address;

	cout<<"Enter the mobile number of the account holder"<<endl;
	cin>>mobile_number;
	cout<<"Enter the age of the account holder"<<endl;
	cin>>age;
	cout<<"Enter your aadhaar number"<<endl;
	cin>>adhar_number;
	int choice;
	cout<<"Enter your choice from the following"<<endl;
	while(true)
	{
		cout<<"1.Savings Account"<<endl;
		cout<<"2.Current Account"<<endl;
		cout<<"3.Exit"<<endl;
		cin>>choice;
		switch(choice)
		{
			case 1:
				cout<<"You have chosen savings account"<<endl;
				break;
			case 2:
				cout<<"You have chosen current account"<<endl;
				break;
			case 3:
				break;
			default:
				cout<<"\nInvalid choice\n";
		}
	break;
	}
	
	
	cout<<"===================*======================"<<endl;
	cout<<"Your account has been successfully created"<<endl;
	cout<<"Thank You for choosing our Bank"<<endl;
	cout<<"===================*======================"<<endl;
	cout<<endl;
}
void BMS::display_account_holder()
{
	cout<<"~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"<<endl;
	cout<<"Account Holder Details"<<endl;
	cout<<"~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"<<endl;
	
	cout<<"===================*======================"<<endl;
	cout<<"Your account number is: "<<acc_holder<<endl;
	cout<<"Your address is: "<<address<<endl;
	cout<<"Your age is: "<<age<<endl;
	cout<<"Your mobile number is : "<<mobile_number<<endl;
	cout<<"Your aadhar number is: "<<adhar_number<<endl;
	cout<<"===================*======================"<<endl;
	cout<<endl;
}
void BMS::deposit()
{
	float total_balance,deposit_amount;
	cout<<"-------------------------"<<endl;
	total_balance = 50000;
	cout<<total_balance<<endl;
	cout<<"--------------------------"<<endl;
	cout<<"Enter the amount to be deposited"<<endl;
	cin>>deposit_amount;
	total_balance += deposit_amount;
	cout<<"********************************************"<<endl;
	cout<<"The account balance is: "<<total_balance<<endl;
	cout<<"********************************************"<<endl;
	cout<<endl;
}
void BMS::withdraw()
{
	int total_balance,withdraw_amount;
	cout<<"---------------------------------"<<endl;
	total_balance = 500000;
	cout<<total_balance<<endl;
	cout<<"---------------------------------"<<endl;
	cout<<"Enter the amount to be withdraw: ";
	cin>>withdraw_amount;
	total_balance -=withdraw_amount;
	
	if(withdraw_amount>total_balance)
		cout<<"You have insufficient balance"<<endl;
		
	else
	{
	cout<<"********************************************"<<endl;
	cout<<"The account balance is: "<<total_balance<<endl;
	cout<<"********************************************"<<endl;
	}

}
void BMS::modifying_account()
{


	cout<<"The present details of the account holder are: "<<endl;
	cout<<"Your account number is: "<<acc_holder<<endl;
	cout<<"Your address is: "<<address<<endl;
	cout<<"Your age is: "<<age<<endl;
	cout<<"Your aadhar number is: "<<adhar_number<<endl;
	
	
	
	cout<<"Enter the details to modify your account"<<endl;
	cout<<"Enter the name of the account holder"<<endl;
	cin>>acc_holder;
	cout<<"Enter the address of the account holder"<<endl;
	cin>>address;
	cout<<"Enter the mobile number of the account holder"<<endl;
	cin>>mobile_number;
	cout<<"Enter the age of the account holder"<<endl;
	cin>>age;
	cout<<"Enter your aadhaar number"<<endl;
	cin>>adhar_number;
	
	cout<<"=================*====================="<<endl;
	cout<<"Modified Details"<<endl;
	cout<<"=================*====================="<<endl;
	
	cout<<"Your name is: "<<acc_holder<<endl;
	cout<<"Your address is: "<<address<<endl;
	cout<<"Your age is: "<<age<<endl;
	cout<<"Your aadhar number is: "<<adhar_number<<endl;
	cout<<endl;
}
void BMS::closing_account()
{
	char reason[30];
	cout<<"Please Explain  the reason for closing the account "<<endl;
	cin>>reason;
	
	cout<<"Please Fill the following details"<<endl;
	cout<<"acc_holder:";
	cin>>acc_holder;
	cout<<"\n mobile_number: ";
	cin>>mobile_number;
	cout<<"\n adhar_number: ";
	cin>>adhar_number;
	cout<<"please wait a minute"<<endl;
	cout<<"Your account has been successfully closed"<<endl;
	cout<<endl;
}

int main()
{
	int ch;
	BMS b;
	do
	{
	
		cout<<"*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*"<<endl;
		cout<<"BANKING MANAGEMENT SYSTEM"<<endl;
		cout<<"*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*"<<endl;
		cout<<"1.Creating an account"<<endl;
		cout<<"2.Displaying the details of the account holder"<<endl;
		cout<<"3.Depositing amount"<<endl;
		cout<<"4.Withdrawing amount"<<endl;
		cout<<"5.Modifying an account"<<endl;
		cout<<"6.Closing an account"<<endl;
		cout<<"7.Exit"<<endl;
		cout<<"Enter your choice"<<endl;
		cin>>ch;
		switch (ch)
		{
			case 1:
				b.create_account();
				break;
			case 2:
				b.display_account_holder();
				break;
			case 3:
				b.deposit();
				break;
			case 4:
				b.withdraw();
				break;
			case 5:
				b.modifying_account();
				break;
			case 6:
				b.closing_account();
				break;
			case 7:
				break;
			default:
			cout<<"Please select the given options"<<endl;
		}
		
	}while(ch!=7);
		return 0;

}


